test/AxelarGateway.js

```diff
@@ -970,10 +971,10 @@ describe('AxelarGateway', () => {
                 const salt = id(`${destinationAddress}-${owner.address}-${getRandomInt(1e10)}`);
                 const depositHandlerAddress = getCreate2Address(gateway.address, salt, keccak256(depositHandlerFactory.bytecode));
 
-                await wallets[0].sendTransaction({ to: depositHandlerAddress, value: '100000000000000000' });
+                await wallets[0].sendTransaction({ to: depositHandlerAddress, value: '100000000000000000' }).then((tx) => tx.wait());
```

Comment:
```suggestion
                await wallets[0].sendTransaction({ to: depositHandlerAddress, value: '1' }).then((tx) => tx.wait());
```
huge value here. can you sanity check other occurrences in all our tests?
instead of wallets[0], refer to a specific wallet?

Reply:
Replaced all instances of wallets[0] with owner for readability. This appears to be the only place where explicit value is passed in with a call, so changed it to '1' as per your suggestion.